### PR TITLE
demo/navigator-interactive-docs

### PR DESCRIPTION
**Pull request by kamil-musialowski**
Added interactive docs to be used on navigator page.

### CODE REVIEW

**Review by DJTechnoo — APPROVED** [review]
Great demos! Approving, with one small remark though: Should the titles of the demos be more descriptive?

**Review by bm64 — APPROVED** [review]

File: samples/stock/interactive-docs/navigator-styling/demo.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        'https://demo-live-data.highcharts.com/aapl-ohlc.json'
+    ).then(response => response.json());
+
+    const chart = Highcharts.stockChart('container', {
+        series: [{
+            type: 'ohlc',
+            id: 'aapl-ohlc',
+            name: 'AAPL Stock Price',
+            data: data
+        }]
```
Comment (pawelfus) [inline samples/stock/interactive-docs/navigator-styling/demo.js:14]: Could you set initial range to be eg 1 year? The first thing I clicked was to change the mask from inner to outer and even though I knew the expected result, it was confusing.

File: samples/stock/interactive-docs/navigator-styling/demo.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" class="chart"></div>
+<div id="button-box">
+    <button id="toggle-mask">Set mask to outside</button><br><br>
+
+    <label for="navigator-mask-fill">Change navigator mask fill:</label>
+    <input type="text" id="navigator-mask-fill"><br><br>
```
Comment (pawelfus) [inline samples/stock/interactive-docs/navigator-styling/demo.html:9]: How about we change the input to a color picker? With a hardcoded opacity (0.3)? I mean, users can change the color, but not the opacity.

**Review by pawelfus — CHANGES_REQUESTED** [review]
Looks great! Minor polishing

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by pawellysy — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**